lib/Github/Api/PullRequest.php

```diff
@@ -14,6 +14,31 @@
  */
 class PullRequest extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    /**
+     * Configure the body type.
+     *
+     * @link https://developer.github.com/v3/pulls/#custom-media-types
+     * @param string|null $bodyType
+     */
+    public function configure($apiVersion = null, $bodyType = null)
```

Comment:
Tell me more about the api version here. That is just to enable preview features, right? You cannot change to an older version of the API. 

Also why have you chosen this order of parameters?